### PR TITLE
Enhances unit capacity tests for volume create

### LINKED ISSUE #1881: Cleanup Volume Create Integration Tests
Capacity unit support is now available and we need to adjust the integration tests to pass different size arguments to through. a

### PR DESCRIPTION
**Pull request by matthewavery**
Adds a test creating volumes of the same size using two different unit types and makes sure they are created successfully and are indeed the same size. 

Fixes #1881

### CODE REVIEW

File: tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot [hunk 1]
```diff
@@ -82,6 +82,28 @@ Docker volume create with specific capacity
     Should Be Equal As Integers  ${rc}  0
     Should Be Equal As Strings  ${disk-size}  96.0G
 
+Docker volume create large volume specifying units
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=unitVol1 --opt Capacity=10G
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Strings  ${output}  unitVol1
+    Set Suite Variable  ${ContainerName}  unitContainer
+    Run  docker ${params} run --name ${ContainerName} -d -v ${output}:/mydata busybox /bin/df -Ph
```
Comment (anchal-agrawal) [inline tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot]: We should grab the exit code and output here, and check that the exit code is 0.

File: tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot [hunk 2]
```diff
@@ -82,6 +82,28 @@ Docker volume create with specific capacity
...
+    ${ContainerRC}  ${output}=  Run And Return Rc And Output  docker ${params} wait ${ContainerName}
+    Should Be Equal As Integers  ${ContainerRC}  0
+    Should Not Contain  ${output}  Error response from daemon
+    ${disk-size}=  Run  docker ${params} logs ${ContainerName} | grep by-label | awk '{print $2}'
+    Should Be Equal As Strings  ${disk-size}  9.5G
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=unitVol2 --opt Capacity=10000
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Strings  ${output}  unitVol2
+    Set Suite Variable  ${ContainerName}  unitContainer2
+    Run  docker ${params} run --name ${ContainerName} -d -v ${output}:/mydata busybox /bin/df -Ph
```
Comment (anchal-agrawal) [inline tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot]: ditto

File: tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot [hunk 3]
```diff
@@ -82,6 +82,28 @@ Docker volume create with specific capacity
...
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Strings  ${output}  unitVol1
+    Set Suite Variable  ${ContainerName}  unitContainer
+    Run  docker ${params} run --name ${ContainerName} -d -v ${output}:/mydata busybox /bin/df -Ph
+    ${ContainerRC}  ${output}=  Run And Return Rc And Output  docker ${params} wait ${ContainerName}
+    Should Be Equal As Integers  ${ContainerRC}  0
+    Should Not Contain  ${output}  Error response from daemon
+    ${disk-size}=  Run  docker ${params} logs ${ContainerName} | grep by-label | awk '{print $2}'
+    Should Be Equal As Strings  ${disk-size}  9.5G
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=unitVol2 --opt Capacity=10000
```
Comment (anchal-agrawal) [inline tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot]: Minor comment - an empty line in-between test "units" would improve readability.

File: tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot (hunk 1 above)
Comment (matthewavery) [inline tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot]: yeah I had this change sitting in my local repo and got side tracked by other issues :P I had a feeling you would catch it.

**Comment by anchal-agrawal** [comment]
LGTM

**Comment by rajanashok** [comment]
lgtm

**Comment by matthewavery** [comment]
♻️

**Pull request merged**